src/bitcoin-cli.cpp

```diff
@@ -433,7 +439,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (conn_type == "addr-fetch") return "addr";
         return "";
     }
-    const int64_t m_time_now{GetTimeSeconds()};
+    const int64_t m_time_now{CountSeconds(Now<CliSeconds>())};
```

Comment:
As this is used solely to measure intervals, it may be another good use for `steady_clock`.

Reply:
Oh, never mind, it compares to the server time.